### PR TITLE
Rename rated rating

### PR DESCRIPTION
**Pull request by pkarman**
connects #7450

### Description
Rename variables per Intake team decision.

NOTE that user-facing text has been left as-is.

### Acceptance Criteria
- [x] Code compiles correctly
- [x] Tests continue to pass

### CODE REVIEW

File: app/models/claim_review.rb [hunk 1]
```diff
@@ -76,7 +76,7 @@ def process_end_product_establishments!
     end_product_establishments.each do |end_product_establishment|
       end_product_establishment.perform!
       end_product_establishment.create_contentions!
-      end_product_establishment.associate_rated_issues!
+      end_product_establishment.associate_rating_request_issues!
```
Comment (shanear) [inline app/models/claim_review.rb]: I think this really is talking about a `rating_issue`

File: app/models/end_product_establishment.rb [hunk 2]
```diff
@@ -187,17 +187,17 @@ def status_active?(sync: false)
     !EndProduct::INACTIVE_STATUSES.include?(synced_status)
   end
 
-  def associate_rated_issues!
-    is_rated = true
-    return if code != source.issue_code(is_rated)
-    return if unassociated_rated_request_issues.count == 0
+  def associate_rating_request_issues!
+    is_rating = true
+    return if code != source.issue_code(is_rating)
```
Comment (shanear) [inline app/models/end_product_establishment.rb]: not related to this PR, but it's very awkward that we hardcode a variable to true, and the only place we use that variable is the line right after 🙃

Comment (pkarman) [reply app/models/end_product_establishment.rb]: Kara and I did that to make it more obvious what the single arg indicated. it was non-obvious looking at it

alternately, we could name it `source.issue_code(is_rating: true)`

Comment (shanear) [reply app/models/end_product_establishment.rb]: Yeah I like that way, even more I like:

`source.issue_code(:rating)` or `source.issue_code(:nonrating)`

File: .circleci/config.yml [hunk 3]
```diff
@@ -262,6 +262,7 @@ jobs:
           path: ~/all_logs.log
 
 workflows:
+  version: 2
```
Comment (jiahuang) [inline .circleci/config.yml]: wait didn't this get removed by Ken's pr here: https://github.com/department-of-veterans-affairs/caseflow/commit/45d39d7c8f7e36fb1e93117921e5cc9a0896f83a

Comment (kencheeto) [reply .circleci/config.yml]: yeah I think that was an attempt to debug the build: https://circleci.com/gh/department-of-veterans-affairs/caseflow/19186

I was following the advice here: https://github.com/CircleCI-Public/config-preview-sdk/blob/master/docs/whats-new.md#whats-new-in-21-configuration

(current docs say that it's required: https://circleci.com/docs/2.0/configuration-reference/#version-1)

It doesn't seem to hurt to leave the version in, and it seems that people get inconsistent results without it, so I think it's fine to re-add here.

**Pull request merged**